jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/AzureFetchOrchestrator.java

```diff
@@ -83,14 +86,15 @@ public Map<String, Object> orchestrate() {
         } catch (Exception e) {
             log.info(e.getMessage());
         }
-        return null;
+        return ErrorManageUtil.formErrorCode();
     }
 
     private List<SubscriptionVH> fetchSubscriptions() {
         List<SubscriptionVH> subscriptionList = new ArrayList<>();
         String accountQuery = "SELECT accountId,accountName,accountStatus FROM cf_Accounts where platform = 'azure'";
         List<Map<String, String>> accounts = rdsdbManager.executeQuery(accountQuery);
-        List<String> tenantList = new ArrayList<>();
+		numberOfAccounts = accounts.size();
```

Comment:
don't like the idea of setting global vars in some random private method
why we cannot use size of result from this method?

also, can you please change logic below - join those two `for` loops into single one?

Reply:
Not addressed.

Reply:
merged two loops into one. 

Number of accounts and subscription list size(returned value) may not be the same. Please see this logic

if (subscriptions.isEmpty()) {
                if (numberOfAccounts > 0) {
                    log.error("Error occurred in job azure-data-collector-job. Not able to connect to any of azure accounts.");
                    AzureDiscoveryJob.jobStatus = "failed";
                }
                ErrorManageUtil.uploadError("all", "all", "all", "Error fetching subscription Info ");
                return ErrorManageUtil.formErrorCode();
            }
            
   we need both values.